lib/datadog/core/utils.rb

```diff
@@ -57,11 +57,16 @@ def self.truncate(value, size, omission = '...'.freeze)
 
       # Ensure `str` is a valid UTF-8, ready to be
       # sent through the tracer transport.
+      #
```

Comment:
Refactor because `options = {}` creates an empty Hash on every call to `utf8_encode`.